src/addons/analog.cpp

```diff
@@ -46,8 +72,30 @@ void AnalogInput::process()
     float deadzone = analogOptions.analog_deadzone / 200.0f;
 
     if ( isValidPin(analogOptions.analogAdc1PinX) ) {
-        adc_select_input(analogOptions.analogAdc1PinX-26); // ANALOG-X
-        adc_1_x = ((float)adc_read())/ADC_MAX;
+        adc_select_input(analogOptions.analogAdc1PinX-26); // ANALOG1-X
```

Comment:
There is a lot of code duplication in this function, you basically do the same for all axes. You should extract the common code into a new function and call this four each axis instead.